### DISCUSSION TITLE
Custom range on chart

### DISCUSSION THREAD

**Discussion by muksidin** [Budibase/budibase#8957] [Help]
Hello, 
I use bar chart, is it possible to show only chart only with its data (for example only on date 2 june)

![Screenshot_20221207_030551](https://user-images.githubusercontent.com/5665115/206012345-351810e2-ecd8-4c52-b2e3-f7417e7c4cd4.png)

**Comment by melohagan** [comment]
Hey @muksidin 

I'm not sure what you're trying to do! What does your data look like?
Sorry for the late response.

**Reply by muksidin** [reply]
only the date which is the data is present. in this case the chart is show the date where there is no data.
